package.json

```diff
@@ -14,11 +14,13 @@
     "lint": "jshint index.js api.js lib test",
     "test": "npm run lint && npm run coverage",
     "test-unit": "mocha -u tdd ./test/setup.js ./test/*.js",
-    "start": "node index.js"
+    "start": "node index.js | ./node_modules/.bin/bunyan"
```

Comment:
Oh, @Hexcles - do you know if I should **not** put the bunyan log parser here? I wasn't sure what gcp goes with a node project (i.e. does it run `npm start`), and if so whether it cares about the output format here. It's possible that the cloud logger just avoids stdout entirely when running in gcp so this may be irrelevant, not sure.

If you don't know, no worries, I'll experiment :)

Reply:
Sorry I'm not sure. I'd guess that this is not necessary but also won't break anything.

Reply:
No worries. That's my guess too, let's find out! :D